dts/riscv/espressif/esp32c3.dtsi

```diff
@@ -21,12 +21,18 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "simple-bus";
-		ranges;
```

Comment:
Is this change related to uart?

Reply:
Oh my bad, it was imported as garbage from other workspace.